specs/Workers.md

```diff
@@ -0,0 +1,1228 @@
+  [propget] HRESULT ServiceWorkerManager([out, retval] ICoreWebView2ServiceWorkerManager** value);
+
+
+  /// Get the shared worker manager to monitor shared worker creations and interact with the shared worker associated with the current profile.
+  ///
+  /// The changes would apply to the context of the user profile. That is, other WebViews under the same user profile could be affected.
+  ///
+  /// \snippet ScenarioSharedWorkerManager.cpp SharedWorkerManager
+  [propget] HRESULT SharedWorkerManager([out, retval] ICoreWebView2SharedWorkerManager** value);
+}
+
+[uuid(4e07e562-8db7-5815-907b-6d89a253a974), object, pointer_default(unique)]
+interface ICoreWebView2ServiceWorkerManager : IUnknown {
+  /// Adds an event handler for the `ServiceWorkerRegistered` event.
+  ///
+  /// A ServiceWorker is a specific type of worker that takes a JavaScript file that can control the web-page/site that it is associated with,
+  /// intercepting and modifying navigation and resource requests, and caching resources in a very granular fashion to give you complete control
+  /// over how app behaves in certain situations.
+  ///
+  /// Service workers essentially act as proxy servers that sit between web applications, the browser, and the network (when available).
+  /// Unlike Shared Workers, which have their own separate global scope, Service Workers have no DOM access and run in a different context.
+  ///
+  /// This event is raised when a web application registers a service worker using the
+  /// `navigator.serviceWorker.register("/sw.js")` method. See the
+  /// [Service Worker Registration](https://developer.mozilla.org/en-US/docs/Web/API/ServiceWorkerRegistration)
+  /// for more information.
+  ///
+  ///
+  /// \snippet ScenarioServiceWorkerManager.cpp ServiceWorkerRegistered
+  HRESULT add_ServiceWorkerRegistered(
+      [in] ICoreWebView2ServiceWorkerRegisteredEventHandler* eventHandler,
+      [out] EventRegistrationToken* token);
+
+  /// Removes an event handler previously added with `add_ServiceWorkerRegistered`.
+  HRESULT remove_ServiceWorkerRegistered(
+      [in] EventRegistrationToken token);
+
+
+  /// Gets a list of the service worker registrations under the same profile.
+  ///
+  /// This method returns a list of `CoreWebView2ServiceWorkerRegistration` objects, each representing a service worker registration.
+  ///
+  /// This method corresponds to the `getRegistrations` method of the `ServiceWorkerContainer` object in the DOM
+  /// which returns a Promise that resolves to an array of `ServiceWorkerRegistration` objects.
+  /// See the [MDN documentation](https://developer.mozilla.org/en-US/docs/Web/API/ServiceWorkerContainer/getRegistrations)
+  /// for more information.
+  HRESULT GetServiceWorkerRegistrations(
+      [in] ICoreWebView2GetServiceWorkerRegistrationsCompletedHandler* handler
+  );
+
+  /// Gets the service worker registration object associated with the specified scope. If a service worker has been registered
```

Comment:
'If a service worker has been registered'... seems like its missing the rest of the sentence